lib/opFns.js

```diff
@@ -288,6 +288,33 @@ module.exports = {
       cb(null)
     })
   },
+  EXTCODEHASH: function (address, runState, cb) {
+    if (!runState._common.gteHardfork('constantinople')) {
+      trap(ERROR.INVALID_OPCODE)
+    }
+    var stateManager = runState.stateManager
+    address = addressToBuffer(address)
+
+    if (runState._precompiled[address.toString('hex')]) {
+      return cb(null, new BN(0))
```

Comment:
I still think this case has `KECCAK256_NULL`. Better to wait for the test cases to appear, otherwise this will be inconsistent.

Reply:
Actually it depends on the state of the account at that address (even if it is a precompile), so if it has a nonce or a balance it is `KECCAK256_NULL` else it is`0`